tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -386,7 +388,7 @@ internal static IApi GetApi(TracerSettings settings, IDogStatsd statsd, Action<D
                     // If this was previously initialized, it will be re-initialized with the new settings, which is fine
                     if (Log.FileLoggingConfiguration is { } fileConfig)
                     {
-                        var logger = LibDatadog.Logger.Instance;
+                        var logger = Logger.Instance;
```

Comment:
Same here, I'd _really_ like to disambiguate which logger we're talking about here

Reply:
fair enough!